common/isMobile.ts

```diff
@@ -1,3 +1,9 @@
-/* eslint-disable */
+import { useWindowSize } from 'common/use-window-size';
 
-export const checkIsMobile = () => window.innerWidth < 780;
+export const checkIsMobile = (innerWidth) => innerWidth < 1150;
```

Comment:
Let's use `window.matchMedia("(max-width: 1150px)");` for dynamicly page resize

Reply:
Then we can not use `useWindowSize` and not listen resize event.

In addition, the `resize` event is quite heavy. That needs throttling

Reply:
Then we can not use `useWindowSize` and not listen `resize` event.

In addition, the `resize` event is quite heavy. That needs throttling

Reply:
I can't responsively react to changes of window width without `addEventListener` somewhere. Below is an example of not listening to 'resize' but still using `eventListener`. Is it an ok approach? I can imagine the situation that width can be changed by rotating of tablet. So it can be useful
<img width="719" alt="Снимок экрана 2023-06-25 в 20 44 27" src="https://github.com/ekaterinburgdev/map/assets/23741246/092e21f3-8a30-41bb-8ce6-95eefa4c365b">
https://betterprogramming.pub/using-window-matchmedia-in-react-8116eada2588

Reply:
Yes, we, of course, need to monitor the screen changes. I didn't specify that we need to handle `matchMedia`'s `change` event, which is much less likely to trigger. :)

Reply:
You have made a custom hook — this is a great DX. Let's hook for matchMedia, as in this example:
https://codesandbox.io/s/usematchmedia-hook-rlfu9?file=/src/hooks/useMatchMediaTS.ts

![image](https://github.com/ekaterinburgdev/map/assets/22644149/b5292fc3-423c-4848-8b85-b033bbf697bc)